examples/google-cloud-speech/output/google-cloud-speech/lib/google/cloud/speech/v1/speech_client_config.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
I think this whole file can go away. The defaults can be specified in gax and there are no knobs in the input configuration to modify these default values.

We'll have to update the client code that depends on this file when we remove it.